Dockerfile

```diff
@@ -1,21 +1,21 @@
-FROM golang:1.17.3-alpine3.14 as builder
+FROM golang:1.17-alpine3.15 as builder
 
 WORKDIR /go/src/github.com/rajatjindal/krew-release-bot
 COPY . .
```

Comment:
without `.dockerignore` it will copy all files including sensitive/unnecessary info such as commit history inside `.git/*`.